configure.ac

```diff
@@ -7085,12 +7085,11 @@ dnl Modules that are not available on some platforms
 dnl AIX has shadow passwords, but access is not via getspent()
```

Comment:
This line still mentions "shadow passwords" (the removed `spwd` is *Shadow Password Database*).

Reply:
Oh right. I also removed it.